build-tools/packages/version-tools/src/internalVersionScheme.ts

```diff
@@ -405,6 +412,25 @@ export function getVersionRange(
 	return range;
 }
 
+function bumpLegacyCompatVersion(
+	version: semver.SemVer | string,
+	bumpRange: number,
+): semver.SemVer {
```

Comment:
Is this function needed to generate the ranges? If not, then I would omit it for now.

Reply:
renamed this fucntion